src/services/cherry-picker.ts

```diff
@@ -102,9 +46,23 @@ export class CherryPicker {
     for (const node of nodes) {
       rawNodes[node.publicKey] = node
       rawNodeIDs.push(node.publicKey)
-      const rawServiceLog = await this.fetchRawServiceLog(blockchain, node.publicKey)
+    }
+
+    // Pull all service & failure & error logs
+    const rawServiceLogs = await this.fetchRawLogs(blockchain, rawNodeIDs, 'service')
+    const rawFailureLogs = await this.fetchRawLogs(blockchain, rawNodeIDs, 'failure')
+    const rawErrorLogs = await this.fetchRawLogs(blockchain, rawNodeIDs, 'errors')
```

Comment:
We can probably optimize this by fetching them all at the same time rather than sequentially, with `Promise.all`

Reply:
Could even be combined into just one trip to redis but since it's Saturday, this is a fine way to test if this is the actual issue